app/cdap/components/Snackbar/Components/Transition/index.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { Box, Typography } from '@material-ui/core';
+import CheckCircleOutlinedIcon from '@material-ui/icons/CheckCircleOutline';
+import CloseIcon from '@material-ui/icons/Close';
+import ErrorOutlineIcon from '@material-ui/icons/ErrorOutline';
+import { addActionType, PREFIX } from 'components/Snackbar/Components/Transition/constants';
```

Comment:
Also these prefixes look like they're only used in this file? Why did you pull them out into a separate file?

Reply:
We have been following maintaining separate constants files for all the constants that we use in any component. In the same way, we have got a constants file for the Transition component as well. So, we tried to move the constants(PREFIX, ADD_ACTION_TYPE) to constants file instead of directly declaring them in the transition component's index fie.